src/rgw/rgw_putobj_processor.cc

```diff
@@ -331,6 +331,11 @@ int AtomicObjectProcessor::complete(size_t accounted_size,
 
   r = obj_op.write_meta(dpp, actual_size, accounted_size, attrs, y);
   if (r < 0) {
+    // if the head object write doesn't succeed, we'll orphan any tail objects as if we'd crashed before that write
+    if (r == -ETIMEDOUT) {
+      // The head object write may eventually succeed, clear the set of objects for deletion
```

Comment:
both of these comments apply to the `ETIMEDOUT` case, can you please put them together?
```c++
// The head object write may eventually succeed, clear the set of objects for deletion. if it
// doesn't ever succeed, we'll orphan any tail objects as if we'd crashed before that write
```

Reply:
already updated